### ISSUE TITLE
Using OIDN in Qt Creator

### ISSUE THREAD

**Issue by Amir-Arsalan** [OpenImageDenoise/oidn#99]
Hi. I am a bit novice in C++ so I apologize in advance if my question is simple.
I want to use ODIN in a program that I am writing/executing in Qt Creator. Should I simply compile OIDN outside Qt Creator and then do `#include <OpenImageDenoise/oidn.hpp>` in my code in order to be able to use OIDN?

**Comment by Amir-Arsalan** [comment]
@atafra I just successfully compiled OIDN and moved the entire directory to where my code is.

Then I tried to initialize a new oidn object for C++11 as follow:
```
oidn::DeviceRef device = oidn::newDevice();
device.commit();
```
But this gives me the following errors:
![image](https://user-images.githubusercontent.com/6061721/106848526-20141980-667f-11eb-8a4a-f1c460d2ba64.png)

I'm not sure why I'm getting those errors. Am I supposed to do something else before executing those two lines?

Note that doing `#include "oidn/include/OpenImageDenoise/oidn.hpp"` works and autocomplete works for oidn. Is this the correct way of including oidn in my codebase?
